### ISSUE TITLE
`pip install image` is required to run the sample

### ISSUE THREAD

**Issue by miparnisari** [Microsoft/Cognitive-Face-Python#41]
I tried to run the sample by following the steps in the readme (installed python 2.7, installed wxPython, installed cognitive_face).

However, I got this error:
```python
C:\Users\mparn\Desktop\Cognitive-Face-Python (master)
λ python sample\
Traceback (most recent call last):
  File "C:\Python27\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "C:\Users\mparn\Desktop\Cognitive-Face-Python\sample\__main__.py", line 8, in <module>
    from view import MyApp
  File "sample\view\__init__.py", line 15, in <module>
    from view.panel_detection import DetectionPanel
  File "sample\view\panel_detection.py", line 10, in <module>
    import util
  File "sample\util.py", line 12, in <module>
    from PIL import Image
ImportError: No module named PIL
```

This is fixed by running `pip install image`.

Although now I get a different error...
```python
λ python sample\
Traceback (most recent call last):
  File "sample\view\__init__.py", line 100, in OnInit
    frame = MyFrame(None)
  File "sample\view\__init__.py", line 80, in __init__
    self.book = MyLabelBook(self)
  File "sample\view\__init__.py", line 31, in __init__
    subscription_panel = SubscriptionPanel(self)
  File "sample\view\panel_subscription.py", line 42, in __init__
    subgridsizer = wx.GridSizer(rows=2, cols=2)
TypeError: GridSizer(): arguments did not match any overloaded call:
  overload 1: 'rows' is not a valid keyword argument
  overload 2: 'rows' is not a valid keyword argument
  overload 3: not enough arguments
  overload 4: not enough arguments
OnInit returned false, exiting...
```

I'm using Python 2.7.14

**Comment by huxuan** [comment]
You should install ‘pillow’ rather tha ‘image’.
There is a ‘requirements.txt’ at the top level of the code base. Just run ‘pip install -r requirements.txt’ in the corresponding folder should get everything prepared except for wxPython 3.0.2.0.

**Comment by miparnisari** [comment]
Oh okay. Should we edit the readme to include this?

**Comment by huxuan** [comment]
Yep, maybe adding an extra item in the sample prerequisite [1] should be ok?
Any comments or suggestions are appreciate.

[1] https://github.com/Microsoft/Cognitive-Face-Python#sample-prerequisite

**Comment by miparnisari** [comment]
Sure, i can create a PR.

**Comment by miparnisari** [comment]
Also, regarding the second error, is the answer i posted here correct? https://stackoverflow.com/questions/46377972/failed-to-run-cognitive-face-python-master-test

I can add that to the PR if it's okay.

**Comment by huxuan** [comment]
That would be more than great! :-)

**Comment by huxuan** [comment]
> Also, regarding the second error, is the answer i posted here correct? https://stackoverflow.com/questions/46377972/failed-to-run-cognitive-face-python-master-test

No, I do not think so. There should have no difference between the code snippets listed in the answer. But do keep in mind we still keep using wxPython 3.0.2.0 as 4.0 is still in beta. I never tested our code base in new released 4.0.

> Sure, i can create a PR.

That would be more than great! :-)

**Comment by miparnisari** [comment]
> No, I do not think so. There should have no difference between the code snippets listed in the answer. But do keep in mind we still keep using wxPython 3.0.2.0 as 4.0 is still in beta. I never tested our code base in new released 4.0.

Ohhh wait I figured it out. When I installed wxPython I ran `pip install wxPython`  and that pulled version 4. To get version 3.0.2 i had to go here: https://sourceforge.net/projects/wxpython/files/wxPython/3.0.2.0/.

Should probably mention that in the readme.

**Comment by huxuan** [comment]
> Ohhh wait I figured it out. When I installed wxPython I ran pip install wxPython and that pulled version 4. To get version 3.0.2 i had to go here: https://sourceforge.net/projects/wxpython/files/wxPython/3.0.2.0/.
> 
>Should probably mention that in the readme.

That make sense to me. Maybe specify the version number should be OK.
Actually, we have an issue for similar topics #30.
